.dockerignore

```diff
@@ -2,3 +2,4 @@ Dockerfile
 index/*
 e2e_data/*
 build/*
+output/*
```

Comment:
This directory does not exist on a fresh clone

Reply:
It could be possible to ignore all but exempt the needed directories using something like  ([.dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file))
```
*
!e2e
!src
!test
!third_party
[...]
```
This could exclude all directories but obviously needs to be tested first as it's just a quick idea.

Reply:
Yes these directories are added by the evaluation and end to end test stuff. I think your latter idea is good. Will try that out now